### DISCUSSION TITLE
Allow habits to occur multiple times per day (ex floss after meals)

### DISCUSSION THREAD

**Discussion by numericOverflow** [iSoron/uhabits#976] [Feature Requests]
Some habits need to happen multiple times per day.  It would be super helpful if I could enable multiple repetitions per day.

A great example is flossing after meals -> ideally this would happen 3 times per day, but when I create a new habit and set the repetition to "Repeat __ 3 __ times in __ 1 __ days"  the app complains and says "You can have at most one repetition per day"

This would be a great change to the app!

**Comment by ke352802081770314** [comment]
You could make it a numerical habit and set the unit "times" goal "3" - it is not quite as convenient because instead of "ticking" you will have to edit the number to increase it by one per repetition. 
If you consider the user interface, which for non-numerical habits only gives you one status per day, what you are trying with your setting cannot be visualized. even if you say M times in N days, the app will not allow you to cuimulate achievements, e.g. if I were to say "have a vegan meal" 5 times per 1 week, and I have two vegan meals on Wednesday, the Wednesday can only count as one achieved occurrence. (I could fake it by booking the second one to a neighboring day that would otherwise have been blank, but that makes it less transparent and modifies the rolling week evaluation)
